interface/elements/msc-profile-view/msc-profile-view.js

```diff
@@ -250,6 +253,88 @@ Polymer({
+      } else if (platform.includes("darwin")) {
+        var pathOfKey = process.env.HOME + '/Library/Musicoin/keystore/';
+      } else if (platform.includes("linux")) { //linux
+        var pathOfKey = process.env.HOME + '/.musicoin/keystore/';
+      }
+    var account = document.getElementById('AccountDialog').textContent;
+    pathOfAccount = Finder.in(pathOfKey).findFiles(account.slice(2));
+    QRCode.toDataURL((fs.readFileSync((String(pathOfAccount)), {encoding: 'utf-8'})), {errorCorrectionLevel: 'H'})
+      .then(result => this.paperImage = result)
+      .catch(error => console.log(error));
+  },
+  exportQRCode: function() {
+    var account = document.getElementById('AccountDialog').textContent;
+    var paperImage = document.getElementById('paper-wallet').src;
+    document.getElementById('fileDialogPaper-' + account).click();
+    document.querySelector('#fileDialogPaper-' + account).addEventListener("change", function() {
+    var tmpPath = document.getElementById('fileDialogPaper-' + account).value;
+    var iconPath = 'file://' + nw.__dirname + '/favicon.png';
+    var alert = {
+       icon: iconPath,
+       body: "As we make QR code from the UTC/JSON file." +
+       " You still need a password to access account"};
+    var imgName  = ('UTC--' + new Date().toISOString() + '--' + account).split(':').join('-');
+    var base64Data = paperImage.replace(/^data:([A-Za-z-+/]+);base64,/, '');
+    var filePath = "";
+    var filePath = tmpPath + '/' + imgName + '.png';
+    fs.writeFile(filePath, base64Data, {encoding: 'base64'});
+    new Notification("QR Code Backup in " + tmpPath, alert);
+    document.getElementById('fileDialogPaper-' + account).value = "";
+  });
+  this.$.paperWalletDialog.close();
+  },
+  clearPaperWallet: function() {
+    document.getElementById('paper-wallet').src = "";
+  },
+  paperWalletImport: function() {
+    this.$.paperWalletImportDialog.open();
+    this.$.importAnyDialog.close();
+  },
+  decodeQRCode: function() {
+    if (process.env.APPDATA != undefined && process.env.APPDATA.includes("Settings")) { //hack for XP
+      var pathOfKey = process.env.APPDATA.slice(0,-17) + '\\AppData\\Roaming\\Musicoin\\keystore\\UTC--';
+      } else if (platform.includes("win32")) {
+        var pathOfKey = process.env.APPDATA + '\\Musicoin\\keystore\\UTC--';
+      } else if (platform.includes("darwin")) {
+        var pathOfKey = process.env.HOME + '/Library/Musicoin/keystore/UTC--';
+      } else if (platform.includes("linux")) { //linux
+        var pathOfKey = process.env.HOME + '/.musicoin/keystore/UTC--';
+      }
+    png = new (require('pngjs').PNG)();
+    document.getElementById('fileDialogPaperImport').click();
+    document.querySelector('#fileDialogPaperImport').addEventListener("change", function() {
+    var PaperWallet = document.getElementById('fileDialogPaperImport').value;
+    png.parse(fs.readFileSync(PaperWallet), function(err, decodedPng) {
+    if(err) {
+      //console.error(err);
```

Comment:
Maybe we should exit here? We don't want stuff falling through if there is an error